ADALiOS/ADALiOS/ADAuthenticationContext+TokenCaching.m

```diff
@@ -95,34 +98,34 @@ - (ADTokenCacheStoreItem*)findCacheItemWithKey:(ADTokenCacheStoreKey*) key
         else
         {
             //We have a cache item that cannot be used anymore, remove it from the cache:
-            [self.tokenCacheStore removeItemWithKey:key userId:userId.userId error:nil];
+            [self.tokenCacheStore removeItemWithKey:key error:nil];
         }
     }
     *useAccessToken = false;//No item with suitable access token exists
-    
-    if (![NSString adIsStringNilOrBlank:key.resource])
-    {
-        //The request came for specific resource. Try returning a multi-resource refresh token:
-        ADTokenCacheStoreKey* broadKey = [ADTokenCacheStoreKey keyWithAuthority:self.authority
-                                                                       resource:nil
-                                                                       clientId:key.clientId
-                                                                          error:&localError];
-        if (!broadKey)
-        {
-            AD_LOG_WARN(@"Unexpected error", localError.errorDetails);
-            return nil;//Recover
-        }
-        ADTokenCacheStoreItem* broadItem = [self extractCacheItemWithKey:broadKey userId:userId error:&localError];
-        if (localError)
-        {
-            if (error)
-            {
-                *error = localError;
-            }
-            return nil;
-        }
-        return broadItem;
-    }
+
+    // TODO: MRRT
+//    if (![NSString adIsStringNilOrBlank:key.resource])
+//    {
+//        //The request came for specific resource. Try returning a multi-resource refresh token:
+//        ADTokenCacheStoreKey* broadKey = [ADTokenCacheStoreKey keyWithAuthority:self.authority
+//                                                                       clientId:key.clientId
+//                                                                          error:&localError];
+//        if (!broadKey)
+//        {
+//            AD_LOG_WARN(@"Unexpected error", localError.errorDetails);
+//            return nil;//Recover
+//        }
+//        ADTokenCacheStoreItem* broadItem = [self extractCacheItemWithKey:broadKey userId:userId error:&localError];
+//        if (localError)
+//        {
+//            if (error)
+//            {
+//                *error = localError;
+//            }
+//            return nil;
+//        }
+//        return broadItem;
+//    }
```

Comment:
The cache itself will now return a token item with a refresh token but no AT if it doesn't find a matching AT in cache. This block of commented out code can be deleted.